src/dicom_parser/header.py

```diff
@@ -299,6 +299,10 @@ def get_data_element(
             Invalid data element identifier
         """
         if isinstance(tag_or_keyword, (tuple, str)):
+            if isinstance(tag_or_keyword, str):
+                tag_or_keyword = (
+                    self.get_private_tag(tag_or_keyword) or tag_or_keyword
+                )
```

Comment:
Added this line to allow fetching "registered" private data elements by keyword (which pydicom doesn't enable).